test/integration/data/pages/_app.js

```diff
@@ -0,0 +1,19 @@
+import React, { Suspense } from 'react'
```

Comment:
What is this test covering, isn't `Suspense` failing?

P. S. I'm completely new to next.js, let alone its development.

Reply:
The data folder covers an experimental feature we've been developing, it's unrelated to anything else.